mono/mini/debugger-agent.c

```diff
@@ -5415,6 +5731,35 @@ ss_bp_add_one (SingleStepReq *ss_req, int *ss_req_bp_count, GHashTable **ss_req_
 	}
 }
 
+#ifdef IL2CPP_MONO_DEBUGGER
+
+static void ss_bp_add_one_il2cpp(SingleStepReq *ss_req, int *ss_req_bp_count, GHashTable **ss_req_bp_cache, Il2CppSequencePointC *sp)
+{
+	// This list is getting too long, switch to using the hash table
```

Comment:
Does this comment still make sense?

Reply:
Yes, I would like to leave it. This is from the hack week debugger, and I think we still could benefit from a different data structure here.